lib/components/app/app-menu.tsx

```diff
@@ -180,11 +180,13 @@ class AppMenu extends Component<
     return (
       <>
         <div
+          aria-expanded={isPaneOpen}
```

Comment:
Might need to associate this component with the sliding pane with `aria-controls` (non-blocking).